lib/fedora_migrate/content_mover.rb

```diff
@@ -1,24 +1,44 @@
 module FedoraMigrate
   class ContentMover < Mover
 
+    include DatastreamVerification
+
+    class Report
+      attr_accessor :name, :mime_type, :original_date, :error
+      def success?
+        error.nil?
+      end
+    end
+
     def migrate
-      return nil_content_message if source.content.nil?
+      return report if nil_source
```

Comment:
Where does the report variable come from?  The method below is result_report and I am not seeing report declared.  I know I must be missing something...